client/src/common/ProtectedRoute.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import { Route, Redirect } from "react-router-dom";
+import { Context as AuthContext } from "contexts/AuthContext";
+
```

Comment:
change protected route to render component prop directly. see [Sample protected route](https://github.com/mssiegel/devnetwork/blob/master/client/src/components/routing/PrivateRoute.js) and [example routes where they're used](https://github.com/mssiegel/devnetwork/blob/master/client/src/components/routing/Routes.js)

Reply:
you just take component out of props and render it directly as  `user ? <Component {...props} />` instead of  `user? render(props)

Reply:
Makes sense. I think this would be cleaner